src/main/java/gregtech/api/unification/material/Material.java

```diff
@@ -370,6 +371,26 @@ public int getBlastTemperature() {
         return prop == null ? 0 : prop.getBlastTemperature();
     }
 
+    @ZenGetter("workingTier")
+    public int getWorkingTier() {
+        return materialInfo.workingTier;
+    }
+
+    @ZenMethod
+    public void setWorkingTier(int workingTier) {
+        if (workingTier < 0) {
+            throw new IllegalArgumentException(
+                    "Cannot set working tier for material " + materialInfo.resourceLocation + "to less than 0 (ULV)!");
+        }
+        if (workingTier > GTValues.MAX_TRUE) {
```

Comment:
There are many places where MAX_TRUE, or even MAX_TRUE - 1 will cause overflows. I think this is best limited to regular MAX, as beyond that is intended for overclocking/parallelism exclusively.